common/src/main/java/org/opensearch/ml/common/transport/mcpserver/requests/McpToolBaseInput.java

```diff
@@ -138,7 +120,13 @@ public XContentBuilder toXContent(XContentBuilder builder, ToXContent.Params xco
             builder.field(PARAMS_FIELD, parameters);
         }
         if (attributes != null && !attributes.isEmpty()) {
-            builder.field(SCHEMA_FIELD, attributes);
+            builder.field(ATTRIBUTES_FIELD, attributes);
+        }
+        if (createdTime != null) {
+            builder.field(CommonValue.CREATE_TIME_FIELD, createdTime.toEpochMilli());
+        }
+        if (lastUpdatedTime != null) {
```

Comment:
what about version?

Reply:
We're not going to expose the version to customer, the version is only for indicating the document has latest update or not to determine if update MCP server memory, so it's for internal use.